### DISCUSSION TITLE
Compilation Instructions

### DISCUSSION THREAD

**Discussion by mrchainman** [FossifyOrg/General-Discussion#599] [Q&A] [answered]
Since not all apps are yet released I would find it helpful to have some cimpilation instructions in the README.

Would that be possible?

**Comment by naveensingh** [comment, marked as answer]
Hi!

Camera will be released probably next week. The current code is exactly the same as Simple Camera so you can just download the last Simple Camera release from here: https://github.com/SimpleMobileTools/Simple-Camera/releases/tag/5.7.2

I'm not sure if you are familiar with command line tools but if you really want to compile the app yourself, easiest way to do that is using Android Studio. There are plenty of tutorials on how to get started Android Studio.

Some links:
 - https://developer.android.com/studio/intro
 - https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository

You can also just download the repository as a zip file if you are not familiar with version control systems like git.
